src/main/cpp/chassis/ChassisOptionEnums.h

```diff
@@ -60,6 +60,19 @@ class ChassisOptionEnums
         HOLONOMIC
     };
 
+    enum AutonChassisOptions
+    {
+        VISIONDRIVENOTE,
+        VISIONDRIVESPEAKER,
+        NONE
```

Comment:
our compiler doesn't allow the same enum value in multiple enums within the same file, so change this one to NO_VISION  

Also, put underscores between the words on the other ones